### ISSUE TITLE
Hard Mode Hammers

### ISSUE THREAD

**Issue by jriwanek** [cyanobacterium/BaseMetals#38]
As requested in http://www.minecraftforum.net/forums/mapping-and-modding/minecraft-mods/2352456-drcyanos-base-metals?comment=325

I'll be poking over this when I've shortened my list a bit so if you beat me to it, it's okay.

**Comment by jriwanek** [comment]
```
Quote from Zygus42»

Hi. We stopped using this excellent mod solely because the presence of the crack hammers was messing up our early game progression. Why bother building a machine to grind powders and a power generation system to run it when you can just slap 9 bits of metal onto some sticks. I came across this today:

disableAllHammers = config.getBoolean("disable_crack_hammer", "options", disableAllHammers,
"If true, then the crack hammer cannot be crafted.");

in BaseMetals/src/main/java/cyano/basemetals/BaseMetals.java

on GitHub/cyanobacterium/BaseMetals

That line does not exist in the latest config. I tried entering the line "disableAllHammers = true" manually into the config but no luck. Is this something you're planning to add in a future version ?
```

**Comment by jriwanek** [comment]
Option to disable all hammers will be implemented in 2.5.0
